### PR TITLE
Multiple ASGs in deployment group

### PR DESCRIPTION
**Pull request by aligentjim**
This PR adds the ability for Moonshot to create and validate deployment groups containing multiple ASGs per my issue #93 ...  This PR also contains the fix from my previous PR ( #95 ).

### CODE REVIEW

**Comment by codwazny** [comment]
Hey, thanks for the contribution. Sorry for not responding earlier but I somehow had this repo silenced :( I think the code looks fine and I really appreciate you keeping it backwards compatible. I want @askreet to take a peek when he gets back from vacation in a few days.

**Comment by aligentjim** [comment]
Great, thanks.  We've been using this change in the wild for a while now and it's been rock solid.  There's no reason it shouldn't be merged.

File: lib/moonshot/deployment_mechanism/code_deploy.rb [hunk 1]
```diff
@@ -203,20 +225,26 @@ def create_deployment_group
       application_name: app_name,
       deployment_group_name: app_name,
       service_role_arn: role.arn,
-      auto_scaling_groups: [asg_name])
+      auto_scaling_groups: asg_names)
   end
 
   def wait_for_asg_capacity
     ilog.start 'Waiting for AutoScaling Group to reach capacity...' do |s|
```
Comment (askreet) [inline lib/moonshot/deployment_mechanism/code_deploy.rb]: Probably want `Autoscaling Group(s)` here as well.

**Comment by askreet** [comment]
Patch looks good to me. Very sorry that this fell under our radar somehow.

File: lib/moonshot/deployment_mechanism/code_deploy.rb (hunk 1 above)
Comment (aligentjim) [inline lib/moonshot/deployment_mechanism/code_deploy.rb]: Agree...  Making that change now.

**Pull request merged**